Login/app/src/test/java/com/projetos/txoza/ValidadeUsers.kt

```diff
@@ -0,0 +1,25 @@
+package com.projetos.txoza
+
+import com.projetos.txoza.util.User
+import org.junit.Test
+import org.junit.Assert.*
+
+class ValidadeUsers {
```

Comment:
Is this class a util to validate user?
Is this class a test?

Your code should be self explanatory